kafka-connect-redis/src/test/scala/com/datamountaineer/streamreactor/connect/redis/sink/config/ConfigMultipleSortedSetsSpec.scala

```diff
@@ -0,0 +1,87 @@
+package com.datamountaineer.streamreactor.connect.redis.sink.config
+
+import com.datamountaineer.streamreactor.connect.redis.sink.support.RedisMockSupport
+import com.datamountaineer.streamreactor.connect.rowkeys.StringStructFieldsStringKeyBuilder
+import org.scalatest.{Matchers, WordSpec}
+import scala.collection.JavaConverters._
+
+/**
+  * Using `SELECT .. FROM .. PK .. STOREAS SS` we can promote the value of one field to a Redis Sorted Set (SS)
+  *
+  * The `score` can:
+  * 1. Be explicitly defined STOREAS SS (score=ts)
+  * 2. If not, try to use the field `timestamp` (if it exists)
+  * 3. If not does not exist use current time as the timestamp <system.now>
+  */
+class ConfigMultipleSortedSetsSpec extends WordSpec with Matchers with RedisMockSupport {
```

Comment:
It's ok. scala test allows us to use *Test or *Spec

the (rule) is to infix with *Spec when u are defining the specs in a test
and *Test when u are actually doing real work and testing it